ext/node/polyfills/tls.ts

```diff
@@ -30,7 +34,23 @@ export function getCiphers() {
   return Object.keys(cipherMap).map((name) => name.toLowerCase());
 }
 
-export const rootCertificates = undefined;
+const certs = {};
+let rootCertificates_;
+function cacheRootCertificates() {
+  rootCertificates_ = ObjectFreeze(op_get_root_certificates());
+}
+ObjectDefineProperty(certs, "rootCertificates", {
+  __proto__: null,
+  configurable: false,
+  enumerable: true,
+  get: () => {
+    // Out-of-line caching to promote inlining the getter.
+    if (!rootCertificates_) cacheRootCertificates();
+    return rootCertificates_;
+  },
+});
+
+export const { rootCertificates } = certs;
```

Comment:
I'm not sure I understand the motivation behind the getter. Aren't we calling it immediately in line 53 here? What are we caching?

Reply:
I'm trying to get this [code](https://github.com/nodejs/node/blob/d7d927844247f30e8288c8b37e9681c6f936c1c3/lib/tls.js#L104-L119) work in deno, but it doesn't as we don't have exports variable.

Reply:
I see, that seems to be only possible when you have an actual object to hang off of instead of a plain variable like with ESM :S